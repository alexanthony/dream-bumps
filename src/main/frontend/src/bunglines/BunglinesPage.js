import React, {PropTypes} from 'react';
import { connect } from 'react-redux';
import {Tabs, Tab} from 'material-ui/Tabs';

import BunglineGenderTabContent from './BunglineGenderTabContent';
import { cancelBuyMember } from '../usercrews/actions';

export const BunglinesPage = ({ cancelBuyMember }) => (
  <Tabs onChange={cancelBuyMember}>
    <Tab label="Men">
      <BunglineGenderTabContent sex="male" />
    </Tab>
    <Tab label="Women">
      <BunglineGenderTabContent sex="female" />
    </Tab>
  </Tabs>
);

BunglinesPage.propTypes = {
  cancelBuyMember: PropTypes.func
}

export default connect(null, { cancelBuyMember })(BunglinesPage);
