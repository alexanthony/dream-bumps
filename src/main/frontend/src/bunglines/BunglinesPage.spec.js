import React from 'react';
import { shallow } from 'enzyme';
import { BunglinesPage } from './BunglinesPage';
import toJson from 'enzyme-to-json';

test('Renders properly', () => {
  const component = shallow(
    <BunglinesPage />
  );
  expect(component).toBeDefined();
  expect(component.find('Tab').length).toEqual(2);
  expect(toJson(component)).toMatchSnapshot()

});
