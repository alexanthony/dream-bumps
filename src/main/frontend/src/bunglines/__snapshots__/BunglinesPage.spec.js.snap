exports[`test Renders properly 1`] = `
<Tabs
  initialSelectedIndex={0}
  onChange={[Function]}>
  <Tab
    label="Men">
    <Connect(BunglineGenderTabContent)
      sex="male" />
  </Tab>
  <Tab
    label="Women">
    <Connect(BunglineGenderTabContent)
      sex="female" />
  </Tab>
</Tabs>
`;
